src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,6 +14,8 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNumber;
+    private static int nextSequenceNumber;
```

Comment:
You need to initialise the value for the nextSequenceNumber. It is not a good practice to leave this kind of variable not initialised.